_configtest.o.d

```diff
@@ -0,0 +1,4 @@
+commandline: -I/home/tmabbot/anaconda3/include/python3.5m -c-I/home/tmabbot/anaconda3/include/python3.5m
```

Comment:
@tyler-abbot You can configure Git to ignore certain files or directories by name. If you add the line `scipy_env/` in your global gitignore file (which you might need to create), Git will pretend the directory `scipy_env`, and everything in it, does not exist. It's really common to do this for files that are automatically generated or that may vary between systems, such as Python environment stuff or compiled code.

Instructions: http://islegend.com/development/setting-global-gitignore-mac-windows/
Official documentation for this feature (way more info than you need): https://git-scm.com/docs/gitignore

To get these files out of your PR without changing your environment, you can use `git rm --cached` on the directory and then amend your original commit: https://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste